lms/static/js/learner_dashboard/collections/program_progress_collection.js

```diff
@@ -1,9 +1,6 @@
-(function(define) {
-    'use strict';
-    define([
-        'backbone'
-    ],
-    function(Backbone) {
-        return Backbone.Collection.extend({});
-    });
-}).call(this, define || RequireJS.define);
+import Backbone from 'backbone';
```

Comment:
Maybe its an unecessary change (I know you were trying to keep it light), but could this just be removed and the place that ProgramProgressCollection is instantiated just be instantiated as a Backbone.Collection?

Reply:
Yeah, I dunno. I'm guessing this was done originally for readability or so we could extend it in the future.

I'm leery of including actual changes along with the porting changes, except where needed. But yeah, maybe this could be cleaned in future.

Reply:
👍